coalib/coala_format.py

```diff
@@ -4,8 +4,8 @@
 from coalib.coala import main as coala_main
 
 
-def main():
-    logging.warning('Use of `coala-format` binary is deprecated, use '
+def main(debug=False):
```

Comment:
This change is also conlicting with https://github.com/coala/coala/commit/fa4abdd13e92c33b85df699c54030d0e44a75e66. Do same for this.